src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,250 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static final int MAX_TASKS = 100;
+    private static int taskCount = 0;
+    static Task[] tasks = new Task[MAX_TASKS];
+
+    static String[] commands = {"todo", "mark", "unmark", "deadline", "list", "help", "event"};
+
+    public static void addTask(Task t) {
+        tasks[taskCount] = t;
+        taskCount++;
+    }
+
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void greet() {
+        printLine();
+        System.out.println("Hello! I'm Duke  U ´ᴥ` U\n" + "What can I do for you?");
+        printLine();
+    }
+
+    public static void goodBye() {
+        printLine();
+        System.out.println("Bye. Hope to see you again soon!ﾉ~");
+        printLine();
+    }
+
+    public static void info() {
+        printLine();
+        System.out.println("This command is not valid, please read through the info and try again :)");
+        System.out.println("Type: [todo] [something], and the system will add a new todo item to your list");
+        System.out.println("Type: [event] [something] /from [when] /to [when], and the system will add an event and the timing");
+        System.out.println("Type: [deadline] [something] /by[when], and the system will add a deadline");
+        System.out.println("Type: [mark] [number], and the system will mark the item of the number as done");
+        System.out.println("Type: [unmark] [number], and the system will unmark the item of the number.");
+        System.out.println("Type: bye, to say goodbye to Duke!");
+        System.out.println("Hope it helps!! woof a nice day ੯•໒꒱❤︎");
+        printLine();
+    }
+
+    public static void process(String s) throws InvalidCommandException{
```

Comment:
```suggestion
    public static void process(String s) throws InvalidCommandException {
```
Slight nitpick but makes it alittle more readable